plane/source/docs/soaring-upcoming.rst

```diff
@@ -0,0 +1,22 @@
+.. _soaring-upcoming:
```

Comment:
```suggestion
.. _soaring-speed-to-fly:
```
need to change file name to this also

Reply:
Need to add a note in common-future-wiki-changes in Plane section for the reference to this page that will be added to the soaring page

Reply:
Ok check now.